sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md

```diff
@@ -20,7 +20,8 @@ irregular-plural-words:
   quota: quota
 
 rename-mapping:
-  LoadTestResource: LoadTesting
+  Resource: LoadTestingBaseResource
+  LoadTestResource: LoadTestingResource
   QuotaResource: LoadTestingQuota
   CheckQuotaAvailabilityResponse: LoadTestingQuotaAvailabilityResponse
   EncryptionProperties: CustomerManagedKeyEncryptionProperties
```

Comment:
QuotaResourceList: LoadTestingQuotaPageList?

Reply:
Similar as above, the suffix is `ListResult` as it used as a result / response of a method.

Reply:
@ArthurMa1978 in that case, should LoadTestResourcePageList be renamed to LoadTestResourceListResult

Reply:
The `LoadTestResourcePageList` is internal class that we don't need to rename, the rename function only works for all public class / interface / method.